conda-recipe/build.sh

```diff
@@ -25,6 +25,10 @@ if [ -n "${TBBROOT}" ]; then
     . ${TBBROOT}/env/vars.sh
 fi
 
+# Set RPATH for wheels
+export CFLAGS="-Wl,-rpath,\$ORIGIN/../dpctl,-rpath,\$ORIGIN $CFLAGS"
```

Comment:
How it will work in Windows?

Reply:
The "build.sh" file is used only for unix build. For windows there should be bld.bat with build commands.
As there is no bld.bat file, windows build now is just fake)